[2024-12-12T12:24:37.706+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-12T12:24:37.721+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: instacart_etl_test.etl_dim_orders manual__2024-12-12T12:21:55.248201+00:00 [queued]>
[2024-12-12T12:24:37.729+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: instacart_etl_test.etl_dim_orders manual__2024-12-12T12:21:55.248201+00:00 [queued]>
[2024-12-12T12:24:37.730+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-12T12:24:37.746+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): etl_dim_orders> on 2024-12-12 12:21:55.248201+00:00
[2024-12-12T12:24:37.757+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'instacart_etl_test', 'etl_dim_orders', 'manual__2024-12-12T12:21:55.248201+00:00', '--job-id', '228', '--raw', '--subdir', 'DAGS_FOLDER/test_etl_dag.py', '--cfg-path', '/tmp/tmptu714anz']
[2024-12-12T12:24:37.758+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1126) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-12T12:24:37.760+0000] {standard_task_runner.py:105} INFO - Job 228: Subtask etl_dim_orders
[2024-12-12T12:24:37.760+0000] {standard_task_runner.py:72} INFO - Started process 1128 to run task
[2024-12-12T12:24:37.813+0000] {task_command.py:467} INFO - Running <TaskInstance: instacart_etl_test.etl_dim_orders manual__2024-12-12T12:21:55.248201+00:00 [running]> on host 8f75106f1023
[2024-12-12T12:24:37.921+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='instacart_etl_test' AIRFLOW_CTX_TASK_ID='etl_dim_orders' AIRFLOW_CTX_EXECUTION_DATE='2024-12-12T12:21:55.248201+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-12T12:21:55.248201+00:00'
[2024-12-12T12:24:37.923+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-12T12:24:37.971+0000] {base.py:84} INFO - Retrieving connection 'mssql_default'
[2024-12-12T12:24:37.971+0000] {base.py:84} INFO - Retrieving connection 'mssql_default'
[2024-12-12T12:24:37.976+0000] {base.py:84} INFO - Retrieving connection 'mssql_default'
[2024-12-12T12:31:49.395+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-12T12:31:50.168+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-12T12:31:50.182+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=instacart_etl_test, task_id=etl_dim_orders, run_id=manual__2024-12-12T12:21:55.248201+00:00, execution_date=20241212T122155, start_date=20241212T122437, end_date=20241212T123150
[2024-12-12T12:31:50.694+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-12T12:31:50.917+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-12T12:31:50.922+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
